measurements/api/measurements.py

```diff
@@ -244,11 +238,11 @@ def list_measurements(
         raise BadRequest("Invalid order")
 
 
-    c_anomaly = func.coalesce(Label.anomaly, Measurement.anomaly, False)\
+    c_anomaly = func.coalesce(Label.anomaly, Measurement.anomaly, false())\
                     .label('anomaly')
-    c_confirmed = func.coalesce(Label.confirmed, Measurement.confirmed, False)\
+    c_confirmed = func.coalesce(Label.confirmed, Measurement.confirmed, false())\
                     .label('confirmed')
-    c_msm_failure = func.coalesce(Label.msm_failure, Measurement.msm_failure, False)\
+    c_msm_failure = func.coalesce(Label.msm_failure, Measurement.msm_failure, false())\
                     .label('msm_failure')
```

Comment:
Wow. Does it really matter if it's `False` or `false()`? It looks like obscure rake if it does.

Reply:
It actually doesn't matter (I realised later). The SQL query it generates however is must more clearer (it will use `false` in the query instead of injecting it via variable substitution).
I added this while I was trying to figure out why I saw many measurements with `anomaly=true && failure=true && confirmed=true` to be sure that it was not an issue with `coalesce`, but figured I may leave it nonetheless as [sqlachemy docs claim](http://docs.sqlalchemy.org/en/latest/core/sqlelement.html#sqlalchemy.sql.expression.false) "true() and false() feature better integrated behavior within conjunctions and on dialects that don’t support true/false constants"